### ISSUE TITLE
Firefox: couldn't load @require from URL file://

### ISSUE THREAD

**Issue by nmaxcom** [Tampermonkey/tampermonkey#1178]
I have been searching for 20 minutes and still don't know what is the best solution to this problem. I have found the bug threads in Firefox, issues in this repo, hints in the documentation... Information is either old, wrong or missing.

Some solutions indicated doing something like what it's shown in the documentation:
<img width="448" alt="image" src="https://user-images.githubusercontent.com/8936090/111260563-58fdc180-8621-11eb-9368-1420ccbfe90b.png">

However this option is not found in TM's settings or Firefox's. 

## Expected Behavior
Tampermonkey loading a local userscript in Firefox using @require, to allow a reasonable development feedback loop, just as I do in Chrome.

## Actual Behavior
Doesn't load, console throws: "Tampermonkey: couldn't load @require from URL file:///Users/.../SearchSelected.user.js"

## Script

(Please give an example of the script if applicable.)

```js
// ==UserScript==
// @name         Search for selected text or open url
// @version      1
// @author       Neithan M.
// @match        *://*/*
// @require      file:///Users/.../SearchSelected.user.js
// @license      MIT
// ==/UserScript==
```

**Comment by derjanb** [comment]
Please see https://github.com/Tampermonkey/tampermonkey/issues/347#issuecomment-269039234

However, I'll add this to the documentation.

**Comment by nmaxcom** [comment]
So am I correct in understanding that requiring a local file is not possible for TM in Firefox?

**Comment by derjanb** [comment]
Yes. This is Firefox's feature request to implement user configurable extension file access: https://bugzilla.mozilla.org/show_bug.cgi?id=1266960
